seed/static/seed/js/services/inventory_service.js

```diff
@@ -116,21 +116,16 @@ angular.module('BE.seed.service.inventory', []).factory('inventory_service', [
      *
      */
 
-    inventory_service.get_property = function (property_id, cycle_id) {
+    inventory_service.get_property = function (view_id) {
       // Error checks
-      if (_.isNil(property_id)) {
-        $log.error('#inventory_service.get_property(): property_id is undefined');
-        throw new Error('Invalid Parameter');
-      }
-      if (_.isNil(cycle_id)) {
-        $log.error('#inventory_service.get_property(): cycle_id is undefined');
+      if (_.isNil(view_id)) {
+        $log.error('#inventory_service.get_property(): view_id is undefined');
         throw new Error('Invalid Parameter');
       }
 
       spinner_utility.show();
-      return $http.get('/api/v2/properties/' + property_id + '/', {
+      return $http.get('/api/v2/properties/' + view_id + '/', {
```

Comment:
the properties v2 API does take the property ID, so how is this working? The v2.1 api is slightly different.

Reply:
i've went through and fixed the remaining items on this. We are now passing view ids everywhere as far as I can tell.